src/webhook_launcher/processes/VCSCOMMIT_QUEUE

```diff
@@ -16,7 +16,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-Ruote.process_definition 'vcsqueue' do
+Ruote.process_definition 'wh_handle_event' do
```

Comment:
I don't really like the handle event, we now have "wh_handle_event", handle_webhook, Payload.handle and the different actions Payload.handle can take.

Also being bitten by "webhook" meaning different things.   How about "wh_process_git_event"?